Vizzuality/gfw
# Feature/welcome-modal

## Overview

Updating the welcome modal with new styles and interactions. 

## Demo

<img width="1225" alt="Screenshot 2020-10-15 at 15 05 40" src="https://user-images.githubusercontent.com/971129/96130236-f086a900-0ef7-11eb-9bd1-98f31a6b46f8.png">

## Notes 

added `explore` key that can be used instead of `promptKey`, as the 3rd option is simply a link to a state, not a tutorial. 

## Testing

- Ensure flows work as specified [here](https://www.notion.so/9dd860f5dfd64475b92f4a3f79f0ff37?v=2098a476c4fa407981718cb4eafad507&p=e01e76259b4e4a8ba62ed4ab1de22571)
- Ensure outro (footer) links work as expected
- Ensure any other modal is not affected by this, also that any other previous flow is working as expected.
